metadata.rb

```diff
@@ -3,8 +3,7 @@
 maintainer_email 'help@sous-chefs.org'
 license          'Apache-2.0'
 description      'Installs/Configures graphite'
-
-version          '1.2.0'
+version          '2.0.0'
 
 supports 'ubuntu'
```

Comment:
because this cookbook uses systemd for service management **only** should this be reflected in the OS supports in the metadata file?

```suggestion
supports 'ubuntu', '>= 16'
supports 'debian', '>= 8'
supports 'redhat', '>= 7'
supports 'centos', '>= 7'
```

i don't know anything about scientific or oracle distros

Reply:
This should be debian >= 9 now

Reply:
Also make sure you're making these floats.